examples/pylint/R0101_too_many_nested_blocks.py

```diff
@@ -1,6 +1,12 @@
-def cross_join(x_list, y_list, z_list):
-    """Perform an all-by-all join of all elements in the input lists,
-    skipping elements which are None.
+from typing import List, Tuple, TypeVar, Optional
+
+T = TypeVar('T')
+
+def cross_join(x_list: List[Optional[T]], y_list: List[Optional[T]],
```

Comment:
Please remove the typevar here (beyond 108/148), and just make this deal with ints.